tests/unit/Str/ReverseTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Tests\Unit\Str;
+
+use PHPUnit\Framework\TestCase;
+use Psl\Str;
+
+final class ReverseTest extends TestCase
+{
+
+    public function provideData(): array
+    {
+        return [
+            ['Hello World', 'dlroW olleH'],
+            ['héllö wôrld', 'dlrôw ölléh'],
+            ['Iñigo Montoya', 'ayotnoM ogiñI'],
+            ['某物', '物某' ],
+            ['что-то', 'от-отч']
+        ];
```

Comment:
we need more test cases here:

```suggestion
        return [
            ['Hello World', 'dlroW olleH'],
            ['héllö wôrld', 'dlrôw ölléh'],
            ['Iñigo Montoya', 'ayotnoM ogiñI'],
            ['某物', '物某' ],
            ['что-то', 'от-отч']
            ['🙂😟', '😟🙂'],
            ['مرحبا', 'ابحرم'],
        ];
```

Reply:
added on separate commit, missing a `,` on the suggestion.